### ISSUE TITLE
Add category and tag clouds to the pattern catalog

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns-vuepress-web#49]
At the moment, the pattern list page is a simple list of patterns. It would be nice to show all the categories and tags as well to make the browsing experience more delightful.

Acceptance criteria

- The patterns page displays the available categories and tags

**Comment by varada610** [comment]
Id like to take this up .But Im not sure exactly what needs to be done.
The patterns page https://java-design-patterns.com/patterns/ needs tags for each pattern listed ?

**Comment by iluwatar** [comment]
Yes, exactly. The most bare-bones approach I can think of is to utilize the existing pages:

- https://java-design-patterns.com/category/
- https://java-design-patterns.com/tag/

We could link these pages to https://java-design-patterns.com/patterns/
